code/test_files/robots.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
I wonder if it's worth reworking this file so that it has less duplication of fields and so that it exercises the consolidation logic thoroughly if testing of dark visitors update is added?

Reply:
I am not entirely sure what you are telling me here. But changing file formats is not my intention here. Maybe elaborate more in a new issue.